apiserver/common/networkingcommon/types_test.go

```diff
@@ -978,6 +978,8 @@ func (s *TypesSuite) TestGetObservedNetworkConfigLoopbackInfrerred(c *gc.C) {
 		ConfigType:    "loopback", // since it is a loopback
 	}, {
 		DeviceIndex:   1,
+		CIDR:          "::1/128",
```

Comment:
Should this be a new section, rather than just adding to this scenario?

I feel like we should handle IPv4-only loopback as well, shouldn't we?